test.sh

```diff
@@ -7,23 +8,29 @@ PACKAGES=$(go list ./... | grep -v /vendor/)
 
 go vet $PACKAGES
 if ! which fgt > /dev/null ; then
-  echo "Please install fgt from https://github.com/GeertJohan/fgt."
-  exit 1
+    echo "Please install fgt from https://github.com/GeertJohan/fgt."
+    exit 1
 fi
 
 if ! which golint > /dev/null ; then
-  echo "Please install golint from github.com/golang/lint/golint."
-  exit 1
+    echo "Please install golint from github.com/golang/lint/golint."
+    exit 1
 fi
 
 for package in $PACKAGES
 do
-        fgt golint ${package##github.com/cloudflare/cfssl/}/*.go
+    fgt golint ${package##github.com/cloudflare/cfssl/}/*.go
 done
 
 # Build and install cfssl executable in PATH
 go install github.com/cloudflare/cfssl/cmd/cfssl
-
 go test $BUILD_FLAGS $PACKAGES
```

Comment:
We seem to be running tests twice here. I moved $BUILD_FLAGS into the `go test` command on line 33 and removed this line without (apparent) issue. Is there a reason to keep this here?